docs/impala/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Impala"
+bookIconImage: ../img/impala-logo.png
```

Comment:
Did we upload the icon somewhere?

Reply:
Yes, the logos are located in the other repo "iceberg-docs", this is the related PR:
https://github.com/apache/iceberg-docs/pull/94